docs/modules/qs.md

```diff
@@ -15,3 +15,9 @@
 ## picoquery
 
 [picoquery](https://www.npmjs.com/package/picoquery) can be used when you need nesting and arrays.
+
+## qs-esm
+[qs-esm](https://www.npmjs.com/package/qs-esm) is an esm-only fork of `qs` with the following core differences:
+
+- no polyfills for ancient Node versions. This reduces the amount of dependencies from 15 to 1, and the bundle size from 11kb to 4.2kb.
```

Comment:
i would reword this slightly to be a little more neutral:

> - no polyfills for legacy Node versions, which reduces bundle size and total dependencies
> - included TypeScript types rather than relying on `@types`

i wouldn't specify the numbers since they will likely change (you'll probably make it smaller one day!)